packages/venia-concept/src/RootComponents/Search/search.js

```diff
@@ -63,7 +63,7 @@ export class Search extends Component {
                     </Query>
                 </small>
                 <Icon
-                    name="x"
+                    name="X"
```

Comment:
You'll see a lot of this in the PR. This name is now just a direct correlation to the name exported by the `react-feather` lib.